src/components/Top-Bar.vue

```diff
@@ -1,56 +1,97 @@
 <template>
-  <oc-navbar id="oc-topbar" tag="header" class="oc-topbar">
-    <oc-navbar-item position="left">
-      <oc-button icon="menu" variation="primary" class="oc-topbar-menu-burger uk-height-1-1" aria-label="Menu" @click="toggleSidebar(!isSidebarVisible)" v-if="!publicPage()" ref="menubutton">
-        <span class="oc-topbar-menu-burger-label" v-translate>Menu</span>
-      </oc-button>
-    </oc-navbar-item>
-    <oc-navbar-item position="center">
-      <router-link to="/" class="oc-topbar-icon">ownCloud X</router-link>
-    </oc-navbar-item>
-    <oc-navbar-item position="right" v-if="!publicPage()">
-      <notifications v-if="activeNotifications"></notifications>
-      <div class="oc-topbar-personal">
-        <avatar class="oc-topbar-personal-avatar" :userid="user.id" />
-        <span class="oc-topbar-personal-label">{{ user.displayname }}</span>
-      </div>
-    </oc-navbar-item>
-  </oc-navbar>
+  <div>
+    <message-bar />
```

Comment:
I'd keep the message bar out of the top bar - just for the sake of keeping it small and simple.
Same for the notifications bell

Reply:
I've now removed the message-bar as it was anyway duplicated.

I'll leave notifications, etc here for now and address its extraction when I get back to the "top bar as web component" topic

Reply:
added TODO in my web component PR: https://github.com/owncloud/phoenix/pull/2620#issuecomment-562344948